recipes/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "related" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0ad2e647da0f382ccc2c24b65562ae74ae849a1b007e74bb09e643ec465a17ce
+
+build:
+  number: 0
+  script: {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - attrs
+    - future
+    - pip
+    - python
+    - python-dateutil
+    - pyyaml
+    - singledispatch
+  run:
+    - attrs
+    - future
+    - python
+    - python-dateutil
+    - pyyaml
+    - singledispatch
+
+test:
+  imports:
+    - related
+  requires:
```

Comment:
You likely don't need the `requires` section since you aren't running any actual test and are just testing the import of the package.